### ISSUE TITLE
Consider defaulting to the highest priority locale in Accept-Language if no locale is provided

### ISSUE THREAD

**Issue by oalders** [reneeb/Mojolicious-Plugin-I18NUtils#8]
Thanks for your work on this.  Would you consider defaulting to using the Accept-Language header if no locale is specifically provided?  This would probably only make sense for the formatters which accept `$locale` as a final arg, but it would make for more succinct calls in the templates.  Something like:

```
sub _locale {
    my $self = shift;
    my $c = shift;
    my $lang
        = HTTP::AcceptLanguage->new( $c->req->headers->accept_language );
    return ( $lang->languages )[0];
}
```

My main concern with this is (as I mentioned) that the current API doesn't always have `$locale` as the final arg, but I thought it was a point worth raising.

**Comment by reneeb** [comment]
This sounds like a great idea. I will have a look at it tomorrow evening (CET).

**Comment by reneeb** [comment]
I just released v0.19 with the proposed changes...

**Comment by oalders** [comment]
:+1: Thanks very much!
